frontend/packages/dev-console/src/components/AddPage.tsx

```diff
@@ -34,29 +34,25 @@ const handleProjectCreate = (project: K8sResourceKind) =>
 
 const EmptyStateLoader: React.FC<EmptyStateLoaderProps> = ({ resources, loaded, loadError }) => {
   const [noWorkloads, setNoWorkloads] = React.useState(false);
-  const knativeData = _.get(resources, ['knativeService', 'data'], null);
+  const daemonSets = _.get(resources, ['daemonSets', 'data']);
```

Comment:
We're moving towards using typescript's optional chaining more instead of lodash get.
```suggestion
  const daemonSets = resources?.daemonSets?.data;
```